CIS560-RecipeManager/CIS560-RecipeManager/Pantry/PantryController.cs

```diff
@@ -12,7 +13,8 @@ public PantryController(IQuery query)
 
         public void LaunchPantryForm()
         {
-            new uiPantry().Show();
+            (pantry = new uiPantry(QueryRepository)).Show();
+            
```

Comment:
I think that instead of passing the an ```IQuery``` to the UI, we should pass delegate methods from the ```PantryController```. 
 We could have an AddIngredient method in PantryController that calls the ```IQuery``` method and give that to ```uiPantry```. 
 Generally, the UI shouldn't know about the backend when possible.